README.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started
+
+**Note**: `sudo` is ommited in this document.
+
+1. You will need to install docker an `docker-compose`. 
+
+    pip install docker-compose
+   
+2. Copy `d0.dsk`, `d1.dsk` and `d2.dsk` into the `data` directory. You will need to obtain separately the disk images containing OpenVMS.
+
+3. Pull the docker image
```

Comment:
No need to pull it before; if the compose file specifies the image, it will be pulled automatically when running.